stories/verifyUser.stories.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import configureStrore from '../src/Redux/store';
+import { Provider } from 'react-redux';
+import VerifyUser from '../src/App/Pages/verifyUser.page';
+import constants from '../src/Redux/constants/index';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+// 2. create the mock
+const mock = new MockAdapter(axios);
+const store = configureStrore();
+
+const withProvider = story => <Provider store={store}> {story()} </Provider>;
+const token = 'pppepepepepppe';
+storiesOf('Verify User Page', module)
+	.addDecorator(withProvider)
+	.add('Success', () => {
+		mock
+			.onGet(`http://eagle-bn-backend-staging.herokuapp.com/api/v1/users/verify/${token}`)
+			.reply(200, { verifyUser: false, status: 200 });
+		return (
+			<Router>
+				<VerifyUser
+					match={{
+						params: {
+							token,
+						},
+					}}
+				/>
+			</Router>
+		);
+	})
+	.add('Error verifying', () => {
+		mock
+			.onGet(`http://eagle-bn-backend-staging.herokuapp.com/api/v1/users/verify/${token}`)
```

Comment:
it will be good if this link is stored in `.env` since we won't always use `staging-link`, someone can choose to use `local` or `production` link